docker/mainnet/nodes/docker-compose.chains.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  bitcoin:
+    image: bitcoin/bitcoin:28.0
+    restart: on-failure
+    ports:
+      - "8332:8332"  # RPC
+      - "8333:8333"  # I2P
+      - "8334:8334"  # P2P
+      - "28332:28332"  # ZMQ
```

Comment:
Should we use `BITCOIN_RPC_PORT` and `BITCOIN_ZMQ_PORT` here?

Reply:
Yes, I just need to check how to do that

Reply:
Done in d19adabf